pupil-hardware/pupil-headset.md

```diff
@@ -28,6 +28,16 @@ The world camera lens are interchangeable, so you can swap between the two lense
   If you change lenses you need to <a href="#camera-intrinsics-estimation">recalibrate the camera</a> to update intrisics. Otherwise 3d calibraiton and accuracy test will not work properly!
 </aside>
 
+#### Arm Extender
+
+> {{< video-webm src="/videos/headset-adjust/arm-extend.webm" >}}
+
+If you need to adjust the eye cameras beyond the built in adjustment range, you can use the orange arm extenders that are shipped with your Pupil headset. The arm extender slides onto the triangular eye camera rail arm triangle rail and adds a new rail further forward and down. This extender works for all existing 120 and 200hz systems.
```

Comment:
If you need to adjust the eye cameras beyond the built in adjustment range, you can use the orange arm extenders that are shipped with your Pupil headset. Unplug your eye camera. Slide the existing eye camera arm off the headset. Slide the arm extender onto the triangular mount rail on the headset frame. Slide the camera onto the extended mount rail. Plug the camera back in. 

The eye camera arm extender works for all existing 120 and 200hz systems.